### PR TITLE
Add Pg16 Support

### LINKED ISSUE #6: Add support for PostgreSQL
Can you please add support for PostgreSQL 16?

### PR DESCRIPTION
**Pull request by Jayko001**
Fix https://github.com/tembo-io/prometheus_fdw/issues/6

### CODE REVIEW

File: .vscode/settings.json
Comment (theory) [inline .vscode/settings.json:1]: I suggest you add this to `.gitgnore`:

```
.vscode/
```

File: Cargo.toml [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
+default = ["pg16"]
+pg14 = ["pgrx/pg14", "pgrx-tests/pg14", "supabase-wrappers/pg14"]
+pg15 = ["pgrx/pg15", "pgrx-tests/pg15", "supabase-wrappers/pg15"]
+pg16 = ["pgrx/pg16", "pgrx-tests/pg16", "supabase-wrappers/pg16"]
 pg_test = []
 
 [dependencies]
 chrono = "0.4.26"
-pgrx = "=0.9.7"
+pgrx = "=0.11.3"
```
Comment (theory) [inline Cargo.toml:18]: Can't wait till supabase-wrappers supports v0.11.4, as it greatly simplifies testing in GitHub workflows.

File: src/lib.rs [hunk 2]
```diff
@@ -64,9 +65,10 @@ fn resp_to_rows(obj: &str, resp: &JsonValue, quals: &[Qual]) -> Vec<Row> {
 }
 
 #[wrappers_fdw(
-    version = "0.1.5",
+    version = "0.2.0",
     author = "Jay Kothari",
-    website = "https://tembo.io"
+    website = "https://tembo.io",
+    error_type = "PrometheusFdwError"
```
Comment (theory) [inline src/lib.rs:71]: Can you have a trailing comma on the last item in the list? Helps prevent weird diffs like this one where the website line had to have a comma added.

If not, I wouldn't hold up this PR to add it, mind you.

**Review by theory — COMMENTED** [review]
Working with rapidly-evolving code is a PITA.

File: .vscode/settings.json
Comment (Jayko001) [reply .vscode/settings.json:1]: already have it https://github.com/tembo-io/prometheus_fdw/blob/main/.gitignore#L6 but idk how it got through the first time

File: src/lib.rs (hunk 2 above)
Comment (Jayko001) [reply src/lib.rs:71]: good call

**Review by theory — APPROVED** [review]

**Pull request merged**